### ISSUE TITLE
Flex suport

### ISSUE THREAD

**Issue by ccorda** [patronage/bubs-next#9]
* Graphql to pull in flex modules, and render the relevant react component for that flex.
* Pass in classes, handle background colors (including when they are repeated)

**Comment by ccorda** [comment]
When talking about classes, this is the sort of thing we've done before to prevent two white sections from having double stacked padding (bottom from first followed by top from bottom)

```
var collapseOnSameColor = function() {
  $('section').each(function (index) {
    var $this = $(this);
    var $next = $this.next();

    /**
     * See if this HTMLElement has padding, if not, there's nothing to remove
     */
    var hasExtraPadding = parseInt($this.css('padding-bottom')) > 0;
    if (!hasExtraPadding) {

      return;
    }

    /**
     * See if this HTMLElement has a <section> sibling, if not, nothing to change
     */
    if ($next.prop('tagName') !== 'SECTION') {

      return;
    }

    /**
     * See if BG colors match
     */
    var thisBgColor = $this.css('background-color');
    var nextBgColor = $next.css('background-color');
    if (thisBgColor === nextBgColor) {
      $this.addClass('padding-bottom-none');
    }
  });
}
```

**Comment by kylehotchkiss** [comment]
Introductory flex support was merged with #39. Should we close this out, or leave it open for further development of the flex sections?

**Comment by ccorda** [comment]
I think we're good here @kylehotchkiss -- we've got a solid starter, and we can always look at later.

One thing I'm curious about is what happens if we have one really heavy flex module that is rarely in use -- would it be more performant to use [dynamic imports](https://nextjs.org/docs/advanced-features/dynamic-import) (either in `flex.js`, or in the `flex/FlexHeavyModule.js` so that we're not loading all of that flex modules dependencies client side:
